retail/2.7/modules/ROOT/pages/sfdc-customization-guide.adoc

```diff
@@ -2,30 +2,429 @@
+
+NOTE: If the option to reset your security token is not available, it is likely that one or more Login IP Ranges are in effect. These can be temporarily removed from the settings page of the profile assigned to the service account user.
+
+== Contact Point Mappings
+
+The following table lists the conditions used to assign ContactPoints in Salesforce:
+
+[%header%autowidth.spread]
+|===
+| CIM PartyRoleType - PartyType | Salesforce Object - Record Type | Condition | Action in Salesforce | Comments
+
+| Customer - Individual
+| Account - PersonAccounts
+| ContactPointAddress with `isUsedForBilling` set as `false` and `isUsedForShipping` set as `false`
+| Assign Address as `PersonMailingAddress`
+| If multiple addresses match this condition, the one with primaryFlag set to true is used; if none set the first one will be used
+
+| Customer - Individual
+| Account - PersonAccounts
+| ContactPointAddress with `isUsedForBilling` set as `true`
+| Assign Address as `BillingAddress`
+|
+
+| Customer - Individual
+| Account - PersonAccounts
+| ContactPointAddress with `isUsedForShipping` set as `true`
+| Assign Address as `ShippingAddress`
+|
+
+| Customer - Individual
+| Account - PersonAccounts
+| ContactPointPhone with `isSMSCapable` set as `false`
+| Assign Phone Number as `Phone`
+| If multiple phones match this condition, the one with primaryFlag set to true is used; if none set the first one will be used
+
+| Customer - Individual
+| Account - PersonAccounts
+| ContactPointPhone with `isSMSCapable` set as `true`
+| Assign Phone Number as `PersonMobilePhone`]
+|
+
+| Customer - Organization
+| Account - Account
+| ContactPointAddress with `isUsedForBilling` set as `true`
+| Assign Address as `BillingAddress`
+| If multiple addresses match this condition, the one with primaryFlag set to true is used; if none set the first one will be used
+
+| Customer - Organization
+| Account - Account
+| ContactPointAddress with `isUsedForBilling` set as `true`
+| Assign Address as `BillingAddress`
+| If multiple addresses match this condition, the one with primaryFlag set to true is used; if none set the first one will be used
+
+| Customer - Organization
+| Account - Account
+| ContactPointAddress with `isUsedFoShipping` set as `true`
+| Assign Address as `ShippingAddress`
+|
+
+| Customer - Organization
+| Account - Account
+| ContactPointPhone with `isFaxCapable` set as `false`
+| Assign Phone Number as `Phone`
+| If multiple phones match this condition, the one with primaryFlag set to true is used; if none set the first one will be used
+
+| Customer - Organization
+| Account - Account
+| ContactPointPhone with `isFaxCapable` set as `true`
+| Assign Phone Number as `Fax`
+|
+|===
+
+*The default value for all flags is `false`.
+
+== Household Record Type
+
+The following record type must be enabled if CIM Customer with Party as `Household` needs to be supported, otherwise this step can be ignored.
```

Comment:
The wording "if CIM Customer with Party" is confusing. Please review for clarity.